test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -70,15 +69,15 @@ declare module 'test-case-reporting' {
       // when we get the build from the database.
       // It is not nullable in the database.
       id?: number;
-      commit_hash: string;
-      pull_request_number: number;
+      commit_sha: string;
+      build_number: number;
       started_at: number;
     }
 
     export interface Job {
       // See comment under `DB.Build.id`
       id?: number;
-      buld_id: number;
+      build_id: number;
       job_number: string;
```

Comment:
Should this be type number?

Reply:
I think so; it's a foreign key which is a number on the database. It doesn't correspond to the build number.

Reply:
Ok, let's change it.

Reply:
This one's already type number

Reply:
Sorry, I was commenting on `job_number`.

Reply:
Oh, sorry, I thought you meant `build_id`. `job_number` is represented as a string in the database because of the `123.456` structure